<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseMode</class>
 <widget class="QWidget" name="ChooseMode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #B4B4B4;
font-family: &quot;Arial&quot;;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="topSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>60</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="styleSheet">
      <string notr="true">font: 24pt; font-weight:700; color: #333333;</string>
     </property>
     <property name="text">
      <string>Please choose mode.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayout">
     <property name="spacing">
      <number>40</number>
     </property>
     <property name="leftMargin">
      <number>100</number>
     </property>
     <property name="rightMargin">
      <number>100</number>
     </property>
     <item>
      <spacer name="leftHSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QFrame" name="gateCard">
       <property name="minimumSize">
        <size>
         <width>484</width>
         <height>407</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame { 
    background: white; 
    border-radius: 20px; 
    padding: 20px;
}
#gateBtn {
    background: transparent;
    border: none;
    font: 14pt; 
    font-weight: 600;
    color: #333333;
}</string>
       </property>
       <layout class="QVBoxLayout" name="vGate">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <spacer name="gTop">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="gateIcon">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>200</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="gateBtn">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>44</height>
           </size>
          </property>
          <property name="text">
           <string>Gate</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="gBottom">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="homeCard">
       <property name="minimumSize">
        <size>
         <width>484</width>
         <height>407</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame { background: white; border-radius: 20px; padding: 20px; }</string>
       </property>
       <layout class="QVBoxLayout" name="vHome">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="leftMargin">
         <number>15</number>
        </property>
        <property name="topMargin">
         <number>15</number>
        </property>
        <property name="rightMargin">
         <number>15</number>
        </property>
        <property name="bottomMargin">
         <number>15</number>
        </property>
        <item>
         <layout class="QHBoxLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="closeHomeBtn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton { 
    border: none; 
    color: #444444; 
    font-weight: 700; 
    font-size: 16pt;
    padding: 0px; 
    background: transparent;
}
QPushButton:hover {
    color: #CB6CE5;
}</string>
            </property>
            <property name="text">
             <string>x</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="hTop">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="homeIcon">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>200</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="homeLabel">
          <property name="styleSheet">
           <string notr="true">font: 14pt; font-weight: 600; color: #333333;</string>
          </property>
          <property name="text">
           <string>Home</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="homeExpandFrame">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>180</height> 
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QFrame { background: transparent; }</string>
          </property>
          <layout class="QVBoxLayout" name="expandLayout">
           <property name="spacing">
            <number>8</number>
           </property>
           <property name="topMargin">
            <number>10</number>
           </property>
           <property name="bottomMargin">
            <number>10</number>
           </property>
           <item>
            <widget class="QLabel" name="enterPrompt">
             <property name="styleSheet">
              <string notr="true">font-size: 11pt; color: #555555; font-style: italic;</string>
             </property>
             <property name="text">
              <string>Please enter home pass</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="hPassLayout">
             <item>
              <spacer name="passLeftSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLineEdit" name="homePassInput">
               <property name="minimumSize">
                <size>
                 <width>260</width>
                 <height>36</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>300</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QLineEdit { 
    border-radius: 18px; 
    padding-left: 15px; 
    padding-right: 15px; 
    background: #CB6CE5; 
    border: 1px solid #CB6CE5; 
    color: white; 
}
QLineEdit::placeholder {
    color: rgba(255, 255, 255, 180); 
    font-style: italic;
}</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="placeholderText">
                <string>Enter home pass</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="passRightSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <spacer>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="forgotPassBtn">
             <property name="styleSheet">
              <string notr="true">
                QPushButton {
                    border: none;
                    background: transparent;
                    color: #333333;
                    font-size: 11pt;
                    text-decoration: underline;
                }
                QPushButton:hover {
                    color: #555555;
                }
              </string>
             </property>
             <property name="text">
              <string>Forgot home pass?</string>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="hBottom">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="rightHSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="bottomSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>120</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>